src/global-table.ts

```diff
@@ -93,6 +99,14 @@ export class GlobalTable extends GlobalTableBase {
       throw new Error('tableName must be non-empty and contain only letters and underscores, ' +
       `got: '${props.tableName}'`);
     }
+    // add current region into table props replicas ?, so the default value would be the current(stack) region
+    this.replicateregions.push({ region: this.env.region });//Token.asString(this.env.region) });
+    // add possible multiple replicas
+    if (props.replicas !== undefined) {
+      props.replicas?.forEach((eachRegion) => {
+        this.replicateregions.push({ region: eachRegion.region });
+      });
+    }
```

Comment:
You are basically checking for undefined twice here, you don't need the outer if statement. The for loop will not execute at all if `replicas` is undefined. 

```suggestion
    props.replicas?.forEach((eachRegion) => {
      this.replicateregions.push({ region: eachRegion.region });
    });
```